course_info/templates/course_info/partials/details.html

```diff
@@ -0,0 +1,211 @@
+{% verbatim %}
+
+<nav>
```

Comment:
We can avoid the load most of the time, assuming the course instance list serialization is sufficient here.  But that sounds like a great idea.  Maybe include it in 2376, since it's already the "followon" change?